variants/rak4631_nomadstar_meteor_pro/platformio.ini

```diff
@@ -0,0 +1,51 @@
+; NomadStar Meteor Pro based on RAK4631 with RGBW LED LP5562 support
+[env:rak4631_nomadstar_meteor_pro]
+extends = nrf52840_base
+board = wiscore_rak4631
+board_check = true
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak4631_nomadstar_meteor_pro -D NOMADSTAR_METEOR_PRO
+  -L "${platformio.libdeps_dir}/${this.__env__}/bsec2/src/cortex-m4/fpv4-sp-d16-hard"
+  ;-DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -DEINK_DISPLAY_MODEL=GxEPD2_213_BN
+  -DEINK_WIDTH=250
+  -DEINK_HEIGHT=122
+  -DRADIOLIB_EXCLUDE_SX128X=1
+  -DRADIOLIB_EXCLUDE_SX127X=1
+  -DRADIOLIB_EXCLUDE_LR11X0=1
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/rak4631_nomadstar_meteor_pro> +<mesh/api/> +<mqtt/>
+lib_deps = 
+  ${nrf52840_base.lib_deps}
+  https://github.com/NomadStar-outdoor/IOBoard-RGB-LP5562-Library.git#9c366c8
+
+; If not set we will default to uploading over serial (first it forces bootloader entry by talking 1200bps to cdcacm)
+; Note: as of 6/2013 the serial/bootloader based programming takes approximately 30 seconds
+;upload_protocol = jlink
+
+; Allows programming and debug via the RAK NanoDAP as the default debugger tool for the RAK4631 (it is only $10!)
+; programming time is about the same as the bootloader version.
+; For information on this see the meshtastic developers documentation for "Development on the NRF52"
+[env:rak4631_nomadstar_meteor_pro_dbg]
+extends = env:rak4631_nomadstar_meteor_pro
+board_level = extra
+
+; if the builtin version of openocd has a buggy version of semihosting, so use the external version
+; platform_packages = platformio/tool-openocd@^3.1200.0
+
+build_flags =
+  ${env:rak4631.build_flags}
+  -D USE_SEMIHOSTING
+
+lib_deps =
+  ${env:rak4631.lib_deps}
+  https://github.com/geeksville/Armduino-Semihosting.git#35b538fdf208c3530c1434cd099a08e486672ee4
```

Comment:
If you replace this with

https://github.com/geeksville/Armduino-Semihosting/archive/35b538fdf208c3530c1434cd099a08e486672ee4.zip

it will build faster.